### ISSUE TITLE
App stops: unrecognized selector sent to instance

### ISSUE THREAD

**Issue by XCarrel** [ionic-team/cordova-plugin-ionic-webview#392]
Working on an app with:
```
Ionic:

   ionic (Ionic CLI)  : 4.12.0 (/Users/Xavier/.nvm/versions/node/v8.11.1/lib/node_modules/ionic)
   Ionic Framework    : ionic-angular 3.9.2
   @ionic/app-scripts : 3.1.6

Cordova:

   cordova (Cordova CLI) : 8.0.0
   Cordova Platforms     : android 7.0.0, ios 4.5.5
   Cordova Plugins       : cordova-plugin-ionic 2.0.4, cordova-plugin-ionic-webview 4.1.1, (and 13 other plugins)

System:

   Android SDK Tools : 26.0.1 (/Users/Xavier/Documents/Android/android-sdk-macosx)
   ios-deploy        : 1.9.4
   NodeJS            : v8.11.1 (/Users/Xavier/.nvm/versions/node/v8.11.1/bin/node)
   npm               : 6.9.0
   OS                : macOS Mojave
   Xcode             : Xcode 10.2.1 Build version 10E1001
```
Root page is LoginPage, login.ts says:
```
@Component({
  selector: "page-login",
  templateUrl: "./login.html",
  styleUrls: ['./login.scss']
})
```

The app built for an iPad on iOS 10.2 fails at startup with:
```
CDVWKWebViewEngine: trying to inject XHR polyfill
-[WKWebViewConfiguration setURLSchemeHandler:forURLScheme:]: unrecognized selector sent to instance 0x15e695a0
*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[WKWebViewConfiguration setURLSchemeHandler:forURLScheme:]: unrecognized selector sent to instance 0x15e695a0'
```

I read a few posts on the - apparently - same issue, but failed miserably at solving it.
Any idea ?

**Comment by jcesarmobile** [comment]
From 3.x, the plugin doesn't support iOS 10, only 11+. We are backporting fixes to 2.x for people needing iOS 10.

The plugin sets the iOS deployment target to 11 here https://github.com/ionic-team/cordova-plugin-ionic-webview/blob/master/plugin.xml#L67, but you probably have the preference too and the value gets overwritten, or you changed it manually or there is a cordova bug not respecting the preference.

**Comment by AcroManiac** [comment]
> From 3.x, the plugin doesn't support iOS 10, only 11+. We are backporting fixes to 2.x for people needing iOS 10.

jcesarmobile, with plugin version 2.3.1 App Store Connect does not publish application and sends error: ITMS-90338: Non-public API usage - The app references non-public selectors in <App Name>: _setAlwaysRunsAtForegroundPriority:.

Is it fixed in 2.5.1?

**Comment by jcesarmobile** [comment]
yes
